### ISSUE TITLE
Question on Paper

### ISSUE THREAD

**Issue by aqaqsubin** [yixinL7/Refactoring-Summarization#2]
Dear Yixin Liu,

I really enjoyed your paper on 2-stage learning framework utilizing the complementarity of different text summarization systems.

I have some questions about paper. 

On your paper, you say as follows. 
Sharing a set of parameters between base and meta system can alleviate the *"Base-Meta learning gap"*.
And, pretrain-then-finetune paradiam can mitigate *"Train-Test distribution gap"*.

But i think, there is no experiment indicating that sharing parameters between base-meta model can boost performance.
In $5.5 and $5.8 experiment, pre-trained Refactor is only used as meta learner. (base model is BART, GSum and PEGASUS)
In $5.6 experiment, pre-trained Refactor is also used as base model, but there is no parameter sharing between the other base model and meta model.

Is there any reason that pre-trained Refactor is not suitable as base model in re-ranking?
And is there performance difference when no parameter sharing with meta model (i.e. Refactor only used as meta model in multi-system) and when sharing parameter between meta and only one base model (in $5.6)?

please correct me if I got it wrong.

Thanks in advance.

**Comment by yixinL7** [comment]
Hi Subin, thank you for your interest in our work! 

> Is there any reason that pre-trained Refactor is not suitable as base model in re-ranking?

For the re-ranking setting, we are using abstractive models to sample different candidate summaries. Refactor is not suitable as a base system in this setting since it is an extractive model.

Parameter sharing between base and meta models is not directly associated with performance improvement. Instead, it is a feature of our refactor model which allows it to work both as a base model and a meta model.  In $5.6, refactor serves as both a strong base model and the meta model. Other summarization models with different architectures will not be able to do this. The advantage of this feature is that we simultaneously get a pre-trained meta model and a strong base model.

Please let me know if you have more questions!

**Comment by aqaqsubin** [comment]
Now i see it.
Thank you for letting me know by taking your time!😃
